### PR TITLE
Changed Angstrom unit to angstrom

### PR DESCRIPTION
**Pull request by krissowat**
first PR !!

### CODE REVIEW

File: tests/saved_test_data/sample_data_model.star [hunk 1]
```diff
@@ -40,7 +40,7 @@ data_model_class_1
 loop_ 
 _rlnSpectralIndex #1 
 _rlnResolution #2 
-_rlnAngstromResolution #3 
+_rlnangstromResolution #3
```
Comment (garrettwrong) [inline tests/saved_test_data/sample_data_model.star]: I think we need to commit this change.

**Review by garrettwrong — CHANGES_REQUESTED** [review]
just one line

File: tests/saved_test_data/sample_data_model.star (hunk 1 above)
Comment (krissowat) [reply tests/saved_test_data/sample_data_model.star]: Fixed.

**Review by garrettwrong — DISMISSED** [review]

File: gallery/tutorials/data/ctffind4/diagnostic_output.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Output from CTFFind version 4.1.14, run on 2023-02-02 15:13:07
 # Input file: test_img.mrc ; Number of micrographs: 1
-# Pixel size: 1.000 Angstroms ; acceleration voltage: 200.0 keV ; spherical aberration: 2.26 mm ; amplitude contrast: 0.07
-# Box size: 1024 pixels ; min. res.: 30.0 Angstroms ; max. res.: 5.0 Angstroms ; min. def.: 5000.0 um; max. def. 50000.0 um
-# Columns: #1 - micrograph number; #2 - defocus 1 [Angstroms]; #3 - defocus 2; #4 - azimuth of astigmatism; #5 - additional phase shift [radians]; #6 - cross correlation; #7 - spacing (in Angstroms) up to which CTF rings were fit successfully
+# Pixel size: 1.000 angstroms ; acceleration voltage: 200.0 keV ; spherical aberration: 2.26 mm ; amplitude contrast: 0.07
```
Comment (janden) [inline gallery/tutorials/data/ctffind4/diagnostic_output.txt]: This is the actual output of the CTFFIND4 tool, so it should probably stay the same. (Same below for `input.txt`.)

File: gallery/tutorials/tutorials/ctf.py [hunk 3]
```diff
@@ -40,9 +40,9 @@
 from aspire.operators import CTFFilter, RadialCTFFilter
 
 radial_ctf_filter = RadialCTFFilter(
-    pixel_size=1,  # Angstrom
+    pixel_size=1,  # angstrom
     voltage=200,  # kV
-    defocus=10000,  # Angstrom, 10000A = 1um
+    defocus=10000,  # angstrom, 10000A = 1um
```
Comment (janden) [inline gallery/tutorials/tutorials/ctf.py]: Spaces between value and units (so `10000 A = 1 um`).

**Review by janden — CHANGES_REQUESTED** [review]
Looks great! Just a few things.

File: gallery/tutorials/tutorials/ctf.py (hunk 3 above)
Comment (janden) [reply gallery/tutorials/tutorials/ctf.py]: Great. There's a second instance of this (just below). Can you fix that one as well?

File: gallery/tutorials/data/ctffind4/diagnostic_output.txt (hunk 2 above)
Comment (janden) [reply gallery/tutorials/data/ctffind4/diagnostic_output.txt]: Thanks. This also needs to be changed for `input.txt`.

**Review by garrettwrong — APPROVED** [review]
Great, thanks!

**Review by janden — APPROVED** [review]
LGTM

**Comment by garrettwrong** [comment]
We're going to disregard the CI jobs failing due to APPLE for the time being. Ready to merge.

**Pull request merged**